rxbinding/src/main/java/com/jakewharton/rxbinding/widget/RxListView.java

```diff
@@ -0,0 +1,29 @@
+package com.jakewharton.rxbinding.widget;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.widget.AbsListView;
+
+import rx.Observable;
+
+import static com.jakewharton.rxbinding.internal.Preconditions.checkNotNull;
+
+public class RxListView {
```

Comment:
This (and the other classes) should also be named `RxAbsListView` since that's the class providing the listener interface.